### ISSUE TITLE
Route circles should have "alt" text

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#12]
"Alt" text is present, but is applied incorrectly to the surrounding elements, rather than the images themselves. Would also be helpful to update alt text to be more than a number. E.g. say "Route 3" instead of just "3"
